### PR TITLE
Add My Animation of night view of Sky(Night Sky)

### PR DESCRIPTION
**Pull request by Chayandev**
Added One more CSS art .
I add my new art Named Night Sky.

### CODE REVIEW

**Comment by Chayandev** [comment]
@virtual @tigerlight @dimor @eyesonly1965 please have a look .

File: include.js [hunk 1]
```diff
@@ -114,11 +114,11 @@ let cards = [
     githubLink: 'https://github.com/MaKloudz'
   },
   {
-    artName: 'That Animation',
-    pageLink: './Art/MaKloudz/index.html',
-    imageLink: './Art/MaKloudz/dat-animation.gif',
-    author: 'Blessing Mutava',
-    githubLink: 'https://github.com/MaKloudz'
```
Comment (sophiabrandt) [inline include.js:121]: It looks like you are deleting the entry from another user? Can you please look into that?

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
See comment regarding `include.js`.

**Comment by Chayandev** [comment]
No actually I just copied there writing format and on that time I pasted it
two times that's why i deleted one , i have not deleted anyone's entey , it
just a mistake , i have updated and corrected that.

**Comment by MattCSmith** [comment]
I have just looked over this and there is indeed 5 lines being removed/replaced by this PR. 
We wont be able to merge in until the other users details are added

**Pull request merged**